integration/tests/cook/util.py

```diff
@@ -1264,3 +1270,16 @@ def _fenzo_fitness_calculator():
 
 def using_data_local_fitness_calculator():
     return _fenzo_fitness_calculator() == 'cook.mesos.data-locality/make-data-local-fitness-calculator'
+
+
+@functools.lru_cache()
+def _supported_isolators():
+    mesos_url = retrieve_mesos_url()
+    slaves = get_mesos_state(mesos_url)['slaves']
+    slave_endpoint = f'http://{slaves[0]["hostname"]}:5051/state.json'
```

Comment:
Same question here about using the existing code for getting the agent host/port

Reply:
Done